report/res/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>HPCC: Visualization</title>
+    <link href="./index.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://unpkg.com/@hpcc-js/common@2.0.0/font-awesome/css/font-awesome.min.css">
+    <script src="https://unpkg.com/@hpcc-js/loader@2.2.6/dist/index.min.js"></script>
+    <script>
+        var espUrl = window.location.href;
+        var debugWuid = window.location.search.split("?")[1];
+        if (debugWuid) {
+            espUrl = "http://play.hpccsystems.com:8010/WsWorkunits/res/" + debugWuid + "/report/res/index.html";
+        }
+        var require = window["@hpcc-js/loader"].unpkg();
```

Comment:
I think this loader will essentially always use the latest packages - may want to switch so its locked to a version?
Also may want to switch to using our CDN server.

Reply:
Is there an example you could link me to showing how the loader can be pointed to our CDN?